aspnet/identity/overview/features-api/two-factor-authentication-using-sms-and-email-with-aspnet-identity/samples/sample14.cs

```diff
@@ -2,7 +2,6 @@
     var userManager = HttpContext.Current.GetOwinContext().GetUserManager<ApplicationUserManager>();
     var roleManager = HttpContext.Current.GetOwinContext().Get<ApplicationRoleManager>();
     const string name = "admin@example.com";
-    const string password = "Admin@123456";
```

Comment:
This variable needs to stay because it's referenced below. How about setting it to an empty string instead?

Reply:
I don't think that will meet security guidelines. The entire article is bad on passwords. It's terrible to say, "don't do this but we're not showing you how to do it securely".

Reply:
Check my change